### PR TITLE
From makefile to webpack

### PR DESCRIPTION
**Pull request by ruslandinov**
To be consistent with wrappers workflow I replaced Makefile, Ruby scripts & Browserify with Webpack.

PT task https://www.pivotaltracker.com/story/show/136684021

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,10 @@ import DashjsWrapper from 'streamroot-dashjs-p2p-wrapper';
 
 First of all, make sure you are using a Node.js version >= 6.0.0
 
-Since the building uses a Ruby script, you need Ruby to be installed on your machine. On most Linux distros and on macOS, it's installed by default, but for Windows you need to install it [manually](https://www.ruby-lang.org/en/).
+Webpack is used for building, so make sure that you have `webpack` installed in your global node binaries otherwise install it like this:
+```
+sudo npm install -g webpack
+```
```
Comment (maxbok) [inline README.md]: No need to install webpack. It's already in the `package.json` file.

File: package.json [hunk 2]
```diff
@@ -17,16 +17,15 @@
...
     "lint": "./node_modules/.bin/eslint lib/ test/",
     "lint-fix": "./node_modules/.bin/eslint --fix lib/ test/",
     "test": "./node_modules/.bin/mocha --require mochahook",
-    "dashjs": "./scripts/dashjs.rb",
-    "wrapper": "./scripts/dist_wrapper.rb",
-    "wrapper_dev": "./scripts/dist_wrapper_dev.rb",
-    "bundle": "./scripts/dist_bundle.rb",
-    "bundle_dev": "./scripts/dist_bundle_dev.rb",
-    "build": "npm run clean && npm run dashjs && npm run wrapper && npm run bundle"
+    "wrapper": "webpack --progress --config webpack/webpack.wrapper.prod.babel.js",
```
Comment (jlouazel) [inline package.json]: On the contrary, you should not use `--progress` for commands that will be trigger by the CI. CircleCI termcaps are very limited and it often give a very ugly and polluted output.

You are free to keep it for `..._dev` commands which will run locally.

File: lib/FragmentLoaderClassProvider.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import Events from '../dashjs/src/core/events/Events.js';
-import MediaPlayerModel from '../dashjs/src/streaming/models/MediaPlayerModel.js';
-import ErrorHandler from '../dashjs/src/streaming/utils/ErrorHandler.js';
-import { HTTPRequest } from '../dashjs/src/streaming/vo/metrics/HTTPRequest';
+import Events from '../node_modules/dashjs/build/es5/src/core/events/Events.js';
```
Comment (jlouazel) [inline lib/FragmentLoaderClassProvider.js]: You better use a shortcut allowed by Node.js.
```js
import Events from 'dashjs/build/es5/src/core/events/Events';
```
This should do the same, but clearer. This is also true for the following ones.

File: README.md (hunk 1 above)
Comment (jlouazel) [inline README.md]: No fan of telling people to globally install packages too. Keeping it locally allows us to control `webpack` version.

File: package.json (hunk 2 above)
Comment (ruslandinov) [inline package.json]: Didn't know that, thanks for telling! Fixed.

File: lib/FragmentLoaderClassProvider.js (hunk 3 above)
Comment (ruslandinov) [inline lib/FragmentLoaderClassProvider.js]: Thanks for telling, fixed.

**Pull request merged**